.config/dotnet-tools.json

```diff
@@ -0,0 +1,12 @@
+{

```

Comment:
this branch seems to have gotten confused as to who is the merge base and who is the merge target...

Reply:
nevermind, i wsa commenting on a merge conflict.